core/src/main/scala/org/apache/spark/rapids/tool/benchmarks/BenchmarkBase.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.rapids.tool.benchmarks
+
+import java.io.{File, FileOutputStream, OutputStream}
+
+/**
+ * This code is mostly copied from org.apache.spark.benchmark.BenchmarkBase
+ *
+ * A base class for generate benchmark results to a file.
+ * For JDK9+, JDK major version number is added to the file names to distinguish the results.
```

Comment:
What would be the files names? Could you provide an example. Thanks!

Reply:
Currently the output result file is named as results.txt and the java version is added as a line inside it - 
`JVM Name                   :   OpenJDK 64-Bit Server VM 
Java Version               :   1.8.0_412 
OS Name                    :   Linux 
OS Version                 :   6.5.0-41-generic 
MaxHeapMemory              :   14219 MB 
Total Warm Up Iterations   :   0 
Total Runtime Iterations   :   1 `
Comment corrected to make this clear